pql.go

```diff
@@ -711,14 +711,17 @@ var knownFunctions struct {
 func initKnownFunctions() map[string]*functionRewrite {
 	knownFunctions.init.Do(func() {
 		knownFunctions.m = map[string]*functionRewrite{
-			"not":       {write: writeNotFunction},
```

Comment:
Review note: sorted this list for personal pleasure. Let me know if y'all would prefer to not do that and keep the previous disorganization.